### PR TITLE
fix: add google tag manager script

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by ahmaxed** [comment]
@scissorsneedfoodtoo, what would be the best way to only include these partials only when the site is in production?

File: src/_includes/partials/gtm-body.njk [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5D6RKKP"
+height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+
+
```
Comment (ahmaxed) [inline src/_includes/partials/gtm-body.njk:4]:
```suggestion
```

Comment (ahmaxed) [inline src/_includes/partials/gtm-body.njk]:
```suggestion
```

**Review by raisedadead — COMMENTED** [review]
What is this waiting on?

**Comment by scissorsneedfoodtoo** [comment]
@raisedadead, I'm taking a look at this now. A review should be up soon.

File: src/_includes/layouts/default.njk [hunk 2]
```diff
@@ -59,6 +59,8 @@
         {% block monetization %}
         {% endblock %}
 
+        {% include "partials/gtm-head.njk" %}
```
Comment (scissorsneedfoodtoo) [inline src/_includes/layouts/default.njk]:
```suggestion
        {% if secrets.eleventyEnv === 'prod' %}
            {% include "partials/gtm-head.njk" %}
        {% endif %}
```

File: src/_includes/layouts/default.njk [hunk 3]
```diff
@@ -133,6 +135,9 @@
     {% else %}
         <body class="home-template">
     {% endif %}
+
+    {% include "partials/gtm-body.njk" %}
```
Comment (scissorsneedfoodtoo) [inline src/_includes/layouts/default.njk]:
```suggestion
    {% if secrets.eleventyEnv === 'prod' %}
        {% include "partials/gtm-body.njk" %}
    {% endif %}
```

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Hey @ahmaxed, thanks again for your patience.

Here's what we can do to only include these partials for production builds:

**Review by scissorsneedfoodtoo — APPROVED** [review]
Tested locally and everything LGTM 👍 👍

**Pull request merged**